Conflict between mysql docker and 03_SqlDB/README.md

Hello !

I just want to point out that there is an error with the mysql docker initialization.
The UMLS_DB_DIR directory (03_SqlDB) is not empty because there is a README.md file in it, which causes the container to abort.

Here are the error lines:
```
2023-01-20T14:12:43.577242Z 0 [ERROR] --initialize specified but the data directory has files in it. Aborting.
2023-01-20T14:12:43.577261Z 0 [ERROR] Aborting
```

The problem was easily solved by deleting 03_SqlDB/README.md
Perhaps it is worth mentioning in the root README file? Or to delete this secondary README file?

By the way, thanks a lot for this great repo and guide!

Alexandre

--- a/03_SqlDB/README.md
+++ b/03_SqlDB/README.md
@@ -1,1 +1,0 @@
-Folder to store contents of MySQL database from Docker Volume. This folder has been added to `.gitignore`.

